Dfe.Academies.Academisation.Data/Repositories/ConversionProjectRepository.cs

```diff
@@ -0,0 +1,148 @@
+﻿using AutoMapper;
+using Dfe.Academies.Academisation.Domain.ApplicationAggregate;
+using Dfe.Academies.Academisation.Domain.ProjectAggregate;
+using Dfe.Academies.Academisation.Domain.SeedWork;
+using Dfe.Academies.Academisation.Domain.TransferProjectAggregate;
+using Dfe.Academies.Academisation.IDomain.ProjectAggregate;
+using Microsoft.EntityFrameworkCore;
+
+namespace Dfe.Academies.Academisation.Data.Repositories
+{
+	public class ConversionProjectRepository : GenericRepository<Project>, IConversionProjectRepository
+	{
+		private readonly IMapper _mapper;
+		private readonly AcademisationContext _context;
+
+		public ConversionProjectRepository(AcademisationContext context, IMapper mapper) : base(context)
+		{
+			_context = context ?? throw new ArgumentNullException(nameof(context));
+			_mapper = mapper;
+		}
+
+		public IUnitOfWork UnitOfWork => _context;
```

Comment:
Why do you need a UoW here?

Reply:
Yeah this is standard with the other repos, can be outside or inside the repository they are all effectively the same instance as the DI is setup to be scoped